__init__.py

```diff
@@ -16,21 +17,21 @@ def __init__(self):
         self.connected = False
         self.wifi_process = None
         self.debug = False  # dev setting, VERY VERBOSE DIALOGS
-        # TODO expose all this in skill settings
-        self.ssid = "OVOS"
+        self.ssid = None
         self.pswd = None
+        self.setup_url = None
         self.grace_period = 45
         self.time_between_checks = 30  # seconds
```

Comment:
these two numbers could also be exposed to settings

Reply:
`self` params moved to settings